rules/backend.yar

```diff
@@ -243,6 +243,36 @@ rule md5_f797dd5d8e13fe5c8898dbe3beb3cc5b {
     strings: $ = "echo(\"FILE_Bad\");"
     condition: any of them
 }
+rule md5_22927e670b0953c01874d626110aa474 {
+    strings:
+        $checkout_page_regex = "new RegExp(\"onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout"
```

Comment:
My last commit on this PR b959172fde481945f8df6be7ef6bfdf6b3d0c093 is actually kind of a duplicate. We have 2 rules closely matching some grelos samples:

```
rule md5_cdn_js_link_js {
    strings: $ = "grelos_v=" // <== makes a full match
    strings: $ = "grelos_v= null"
    condition: any of them
}
```
and 

```
rule md5_22927e670b0953c01874d626110aa474 {
    strings:
      // $ =  /(RegExp[^\)]*(onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout)[^\)].+(grelos))/ <== makes a full match
  $ =  /(RegExp[^\)]*(onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout)[^\)].+(grelos_v=null))/
    condition: any of them
}
```
on at least these samples

* malware/backend/22927e670b0953c01874d626110aa474
* malware/frontend/view-source_https___store.latimes.com_onepage_.html
* malware/frontend/view-source_https___store.chicagotribune.com_onepage_.html

I think we should merge 22927e670b0953c01874d626110aa474 into md5_cdn_js_link_js as it covers all and still has a pretty broad detection for the checkout regex. Could you verify the performance is good enough. I'm okay with making md5_cdn_js_link_js less specific to just grelos_v. It's pretty widespread.